cli/wallet/validator.go

```diff
@@ -71,14 +71,14 @@ func newValidatorCommands() []cli.Command {
 }
 
 func handleRegister(ctx *cli.Context) error {
-	return handleCandidate(ctx, "registerCandidate")
+	return handleCandidate(ctx, "registerCandidate", 1001*100000000) // registering costs 1000 GAS
```

Comment:
Why did `-1` stop working?

Reply:
We're test-running via RPC and it only has 15 GAS for execution. Making this limit 1000+ is not something we want.